test/TEST-02-SYSTEMD/test.sh

```diff
@@ -97,26 +62,15 @@ test_setup() {
     grep -U --binary-files=binary -F -m 1 -q dracut-root-block-created "$TESTDIR"/marker.img || return 1
     rm -- "$TESTDIR"/marker.img
 
-    (
-        # shellcheck disable=SC2031
-        export initdir=$TESTDIR/overlay
-        # shellcheck disable=SC1090
-        . "$basedir"/dracut-init.sh
-        inst_multiple poweroff shutdown dd true
-
-        inst_hook shutdown-emergency 000 ./hard-off.sh
-
-        # systemd-analyze.sh calls man indirectly
-        # make the man command succeed always
-        inst "$(find_binary true)" "/usr/bin/man"
-        inst_hook pre-pivot 000 ./systemd-analyze.sh
-
-        inst_hook emergency 000 ./hard-off.sh
-    )
+    # systemd-analyze.sh calls man indirectly
+    # make the man command succeed always
     "$basedir"/dracut.sh -l -i "$TESTDIR"/overlay / \
-        -a "debug systemd" \
+        -a "test systemd" \
         -o "network kernel-network-modules" \
         -d "piix ide-gd_mod ata_piix ext3 sd_mod" \
+        -I "dd" \
```

Comment:
The `test` module depends on `debug`, which installs `dd`, so I think this line is not needed.